src/eva/data/eva_interactive.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python
+
+# (C) Copyright 2023 United States Government as represented by the Administrator of the
+# National Aeronautics and Space Administration. All Rights Reserved.
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+import os
+import netCDF4 as nc
+import hvplot.pandas
+import xarray as xr
+import re
+
+from eva.data.data_collections import DataCollections
+from eva.utilities.logger import Logger
+from eva.utilities.timing import Timing
+from eva.eva_base import EvaFactory
+from eva.transforms.arithmetic import arithmetic
+from eva.transforms.accept_where import accept_where
+# --------------------------------------------------------------------------------------------------
+
+
+class EvaInteractive():
+
+    def __init__(self, filename):
+        self.data_collection = DataCollections()
+        self.logger = Logger('EvaInteractive')
+        self.collection = 'interactive'
+        self.timer = Timing()
+        self.filename = filename
+        self.var_cache = []
+
+    def load_data_collection(self):
```

Comment:
Can you rename this to `load_ioda` or something like that? Since it's hardcoded underneath.